sympy/physics/control/lti.py

```diff
@@ -2205,7 +2211,7 @@ def __neg__(self):
         return MIMOParallel(*arg_list)
 
 
-class Feedback(SISOLinearTimeInvariant):
+class Feedback(TransferFunction):
```

Comment:
I'm not sure about it. Also making changes to single input single output systems but not modifying multiple input multiple output systems seem odd and confusing to me.
https://github.com/sympy/sympy/blob/46a41c0d706f75cf44a920e2a2fd0f2e8b8f8298/sympy/physics/control/lti.py#L2503-L2506

Reply:
For MIMO systems feedback we have to use ` TransferFunctionMatrix` as parent class , but according to https://github.com/sympy/sympy/issues/26161 this says only to implement `Feedback` as a `TransferFunction`.

Reply:
I've tried using `TransferFunctionMatrix` as a parent class of `MIMOFeedback` but there it shows it as undefined 
![image](https://github.com/sympy/sympy/assets/140839576/efd1a2e8-d06f-465a-8907-5ecb5c9c25a5)
```
>>> from sympy.physics.control.lti import *
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "E:\Sympy\sympy\sympy\physics\control\__init__.py", line 1, in <module>
    from .lti import (TransferFunction, Series, MIMOSeries, Parallel, MIMOParallel,
  File "E:\Sympy\sympy\sympy\physics\control\lti.py", line 2537, in <module>
    class MIMOFeedback(TransferFunctionMatrix):
                       ^^^^^^^^^^^^^^^^^^^^^^
NameError: name 'TransferFunctionMatrix' is not defined. Did you mean: 'TransferFunction'?
```
Also it is not the problem of the IDE

Reply:
Unresolving this. I would like this be addressed before we think about merging.